consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/converter/XmlToObjectConverter.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.moj.sdt.cmc.consumers.converter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import java.io.IOException;
+import org.springframework.stereotype.Component;
+
+@Component
+public class XmlToObjectConverter {
+
+    public <T> T  convertXmlToObject(String xml,
```

Comment:
Double space between "T" and "convertXmlToObject.

Reply:
done